examples/minimal-logs/README.md

```diff
@@ -0,0 +1,31 @@
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
Let's add a description of the example in the README

Reply:
Done.